gramps_webapi/api/resources/filters.py

```diff
@@ -132,13 +132,15 @@ def build_filter(filter_parms: Dict, namespace: str) -> GenericFilter:
     return filter_object
 
 
-def apply_filter(db_handle: DbReadBase, args: Dict, namespace: str) -> List[Handle]:
+def apply_filter(
+    db_handle: DbReadBase, args: Dict, namespace: str, handles: List = None
```

Comment:
`List=None` is not a valid type hint. Also, I suggest never to use a bare `List`. In this case, probably `Optional[List[Handle]]`.

Reply:
I can change it.